lib/protocol/Result.js

```diff
@@ -76,41 +76,36 @@ Result.prototype.getLobColumnNames = function getLobColumnNames() {
 };
 
 Result.prototype.handle = function handle(err, reply, cb) {
-  if (err) {
-    return cb(err);
-  }
+  var reply = reply || {};
```

Comment:
I don't like the idea of having FunctionCodes at multiple locations and having to make changes at multiple places for a single FunctionCode. So if something changes for FunctionCode.xyz it's just one place to change. 

Also if we adopt the above example, we set ourselves up  to keep adding single condition statement above the main switch statement.